astronomer/astronomer
# Dev updates 2024-11-12

## Description

Dev workflow updates:
- Use new CircleCI machine image 2024.08.1
- Use latest version of kind and kind images
- Use latest version of helm, gcloud
- Rebuild python test dependency list
- Add `ruff-format` pre-commit hook and apply its suggestions. (This should've been added when we removed `black`)

## Related Issues

NA, KTLO work

## Testing

If tests pass then we're good. None of these changes are included as code that runs in the product that is shipped to customers.

## Merging

Merge everywhere
